db/seeds.rb

```diff
@@ -49,3 +49,10 @@
     world_location_type_id: 1,
   )
 end
+
+if WorldLocation.where(name: "Test International Delegation").blank?
```

Comment:
Bit surprised to see this in this PR as it doesn't seem related to the trello request but it looks fine.